src/meep.hpp

```diff
@@ -17,7 +17,9 @@
 #ifndef MEEP_H
 #define MEEP_H
 
+#include <functional>
```

Comment:
Out of curiosity, what are we using from this header?

Reply:
std::hash